packages/generator/src/parser/parser-util.ts

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+export function stripNamespace(name: string): string {
+  const nameParts = name.split('.');
+  return nameParts[nameParts.length - 1];
+}
+
+const collectionRegExp = /Collection\((?<collectionType>.*)\)/;
+
+export function isCollection(typeName: string): boolean {
+  return collectionRegExp.test(typeName);
+}
+
+export function parseTypeName(typeName: string): string {
+  return isCollection(typeName) ? parseCollectionTypeName(typeName) : typeName;
+}
+
+export function parseCollectionTypeName(typeName: string): string {
```

Comment:
Why this is exported?

Reply:
I intended to use it directly, but then didn't. I can remove the export.